src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -132,3 +198,47 @@
             :on-change      on-tab-change
             :data           tabs-data}]
           [tabs/view {:selected-tab @selected-tab}]]]))))
+
+(defn view
+  [_]
+  (let [title-ref          (atom nil)
+        set-title-ref      #(reset! title-ref %)
+        title-bottom-coord (atom 0)
+        {:keys [top]}      (safe-area/get-insets)
+        set-title-bottom   (fn [_ y _ height]
+                             (reset! title-bottom-coord
+                               (+ y height -100 (if platform/ios? (- top) top))))]
+    (fn []
+      (let [theme                      (quo.theme/use-theme)
+            scroll-amount              (reanimated/use-shared-value 0)
+            title-opacity              (reanimated/use-shared-value 0)
+            collectible                (rf/sub [:wallet/last-collectible-details])
+            {:keys [preview-url
+                    collection-data
+                    collectible-data]} collectible
+            {preview-uri :uri}         preview-url
+            {collectible-name :name}   collectible-data
+            {collection-name :name}    collection-data]
+
+        (rn/use-unmount #(rf/dispatch [:wallet/clear-last-collectible-details]))
+
+        [rn/view
+         {:style {:flex             1
+                  :background-color (colors/theme-colors colors/white colors/neutral-95 theme)}}
```

Comment:
Can this live in the `style.cljs` file?
It is mostly a nit, but we only allow one-liners for styles in the view level.

Reply:
done